docs/tools/vdb_table/data/couchbase.json

```diff
@@ -0,0 +1,181 @@
+{
+  "name": "Couchbase",
+  "links": {
+    "docs": "https://docs.couchbase.com/cloud/vector-search/vector-search.html",
+    "github": "https://github.com/couchbase",
+    "website": "https://www.couchbase.com/products/vector-search/",
+    "vendor_discussion": "https://github.com/superlinked/VectorHub/discussions/xx",
+    "poc_github": "https://github.com/1tylermitchell",
+    "slug": "couchbase"
+  },
+  "oss": {
+    "support": "partial",
+    "source_url": "https://www.couchbase.com/blog/couchbase-adopts-bsl-license/",
+    "comment": ""
+  },
+  "license": {
+    "value": "Proprietary",
+    "source_url": "https://www.couchbase.com/blog/couchbase-adopts-bsl-license/",
+    "comment": "Vector search currently only part of Enterprise and DBaaS editions"
+  },
+  "dev_languages": {
+    "value": [
+      "Python", "c++", "erlang", "go", "java"
+    ],
+    "source_url": "https://docs.couchbase.com/home/sdk.html",
+    "comment": "SDK client support: Python, C++, C, Go, Java, Kotlin, Node, PHP, Ruby, Scala, .Net"
+  },
+  "vector_launch_year": 2024,
+  "metadata_filter": {
+    "support": "full",
+    "source_url": "https://www.prnewswire.com/news-releases/couchbase-announces-new-features-to-accelerate-ai-powered-adaptive-applications-for-customers-302074941.html",
```

Comment:
this needs to be a link to the documentation where it is shown how to filter on various attributes (<,>,== etc.)